spec/helpers/login_helper.rb

```diff
@@ -0,0 +1,10 @@
+module LoginHelper
```

Comment:
Shubham has already written one api headers support helper. how many we are going to create

Reply:
Ok as disucssed - sticking with Shubham's approach.